shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/job/progress/JobProgress.java

```diff
@@ -46,19 +41,17 @@ public final class JobProgress implements PipelineJobProgress {
     
     private boolean active;
     
-    private Map<String, InventoryTaskProgress> inventoryTaskProgressMap;
+    private InventoryTaskProgress inventory;
     
-    private Map<String, IncrementalTaskProgress> incrementalTaskProgressMap;
+    private IncrementalTaskProgress incremental;
     
     /**
-     * Get incremental position.
-     *
-     * @param dataSourceName data source name
+     * get incremental position.
+     * @param dataSourceName dataSource
      * @return incremental position
      */
```

Comment:
Javadoc first character should be upper case. And it's better to leave a blank line better parameters.